tests/unit/DefineCompany.spec.ts

```diff
@@ -63,7 +63,7 @@ for (const mock of mockEntityInfo) {
       expect(wrapper.find('#folio-number-header').text()).toContain('Folio / Reference Number (optional)')
     })
 
-    itIf(mock.entityType !== 'BC')(`displays company statement`, () => {
+    itIf(['BEN', 'ULC', 'CCC'].includes(mock.entityType))(`displays company statement`, () => {
```

Comment:
Yes! Good change! This includes the specified types instead of assuming that everything not the other type is OK.